### PR TITLE
Todo-13-BE: As a user, I want to see all users arranged alphabetical

### PR DESCRIPTION
**Pull request by itc02**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -41,11 +41,11 @@ def get_all_users
   end
 
   def get_users
-    User.select("users.id, users.user_name")
+    User.select("users.id, users.user_name").order(:user_name)
```
Comment (Deathlive) [inline app/controllers/users_controller.rb:44]: @itc02
:thumbsup:

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -41,11 +41,11 @@ def get_all_users
   end
 
   def get_users
```
Comment (Deathlive) [inline app/controllers/users_controller.rb:43]: @itc02 
Es methodnen `get_users` and `user_todos_number` hunch chukechi Privata.
Lav kini orinak obshi `private` tanq.

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -41,11 +41,11 @@ def get_all_users
...
-    User.select("users.id, users.user_name")
+    User.select("users.id, users.user_name").order(:user_name)
   end
 
   def user_todos_number
-    User.all.collect do |user|
+    get_users.collect do |user|
       TodoList.where(:user_id => user.id).count
```
Comment (Deathlive) [inline app/controllers/users_controller.rb:49]: 1) Qani ver stx prosto `count` stanumnq, `order` petqa ?

2) Mindel karinq sti aninq `TodoList.where(:user_id => user.id).count` > `user.todo_lists.count`.

**Review by Demigoddd — APPROVED** [review]

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (itc02) [inline app/controllers/users_controller.rb:50]: 1. `render`-i `todos_number` - y, ver uxarkvuma client side, ta integerneri arraya, vortex amen integer asuma, ta amen user qani todo one. Isk `render`-i `users`-y ta lox usernenenn elya arrayi mech. Meheng, `todos_number`-i 0-errord indexi tivy asuma, ta `users`-i 0-errord indexi usery qani todo one, `todos_number`-i 1-in indexi tivy asuma, ta `users`-i 1-in indexi usery qani todo one, ev aydpes sharunak. Tra heti `order`-y ogtagorcalum.
2. Usery `todo_lists` column chi one, ver ti anim.

Comment (Deathlive) [reply app/controllers/users_controller.rb:50]: @itc02 
1) Uremns piti indexy poxis, ver srazu todos_numberav get anve.
**TODO-10** PR-umy commentum kiral et pahavu

2) Qani ver `orders` anum erevi et patcharov stachvumchi

**Pull request closed without merging**